Sources/TSCBasic/Process.swift

```diff
@@ -393,9 +450,10 @@ public final class Process {
         }
 
         // Print the arguments if we are verbose.
-        if self.verbose {
-            stdoutStream <<< arguments.map({ $0.spm_shellEscaped() }).joined(separator: " ") <<< "\n"
-            stdoutStream.flush()
+        if let loggingHandler = self.loggingHandler {
+            loggingHandler(arguments.map({ $0.spm_shellEscaped() }).joined(separator: " "))
```

Comment:
`\n` not needed?

Reply:
not any more, it was for the stream itself. the handler will decide what to do with it now